vmdb/app/models/ext_management_system.rb

```diff
@@ -281,7 +281,7 @@ def resource_pools_non_default
     if association_cache.include?(:resource_pools)
       self.resource_pools.select { |r| !r.is_default }
     else
-      self.resource_pools.all(:conditions => ["is_default != ?", true])
+      self.resource_pools.where("is_default != ?", true).to_a
```

Comment:
I think this is actually a bug in Arel or maybe AR::Relation (that is, querying a boolean column with a literal true or false).  I was trying to write a query for this the other day in a similar fashion and it doesn't return what is expected.  I had meant to put together a bug report, and I totally forgot.